django/db/models/functions/json.py

```diff
@@ -0,0 +1,23 @@
+from django.db.models.expressions import Func, Value
```

Comment:
I'm not sure if we need a separate `json.py` module. I would include it in `comparison.py` like `Cast()`, _"Comparison and conversion functions"_.

Reply:
I don't see many candidates for this module. Adding many database functions for JSON fields is controversial for me, we didn't have many (if any) requests for them to the previous implementation in the `contrib.postgres` package. TBH I wasn't even convinced about `JSONObject`, but OK lets have it.